micromagneticmodel/dynamics/slonczewski.py

```diff
@@ -29,9 +29,9 @@ class Slonczewski(DynamicsTerm):
     .. math::
 
         \frac{\text{d}\mathbf{m}}{\text{d}t} =
-        -\beta\frac{\epsilon - \alpha\epsilon'}{1+\alpha^{2}}
+        - |\gamma| \beta\frac{\epsilon - \alpha\epsilon'}{1+\alpha^{2}}
```

Comment:
According to me the new equation should be $- \gamma \beta \frac{(\epsilon + \alpha \epsilon')}{1 + \alpha^2}(\mathbf{m} \times (\mathbf{m} \times \mathbf{m_{\text{p}}})) - \gamma \beta \frac{\epsilon' - \alpha \epsilon}{1 + \alpha^2}(\mathbf{m} \times \mathbf{m_{\text{p}}})$.